lib/logstash/codecs/avro.rb

```diff
@@ -59,6 +59,14 @@ class LogStash::Codecs::Avro < LogStash::Codecs::Base
 
   include LogStash::PluginMixins::EventSupport::EventFactoryAdapter
 
+  BINARY_ENCODING = "binary"
+  BASE64_ENCODING = "base64"
```

Comment:
nitpick: strings are mutable, so if we are constantizing them we should also be freezing them.

```suggestion
  BINARY_ENCODING = "binary".freeze
  BASE64_ENCODING = "base64".freeze
```